src/blocks/SingleEvent/config.ts

```diff
@@ -19,7 +19,7 @@ const defaultFields: Field[] = [
 const singleEventWithFields = (fields?: Field[]): Block => ({
   slug: 'singleEvent',
   interfaceName: 'SingleEventBlock',
-  imageURL: '/thumbnail/SingleEventThumbnail.jpg',
+  imageURL: '/thumbnail/SingleBlogPostThumbnail.jpg',
```

Comment:
Same here. this should be 
```suggestion
  imageURL: '/thumbnail/SingleEventThumbnail.jpg',
```